### DISCUSSION TITLE
How to stage all files with a given extension using the magit interface?

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [magit/magit#5165] [Support] [answered]
I know this is possible by executing:

```
$ find . -type f -name '*.srt' -exec git add {} +
```

or 

```
$ git add *.srt
```

I was wondering if it is possible to do this using the magit interface to avoid using the terminal (thus, avoiding [switch cost](https://en.wikipedia.org/wiki/Task_switching_(psychology)#Switch_cost))

**Comment by tarsius** [comment, marked as answer]
No, you would have to write your own command for that.
